### ISSUE TITLE
Requesting support for TF2.3.x

### ISSUE THREAD

**Issue by dspyrhsu** [sicara/tf-explain#156]
When trying to run GradCAM with TF 2.3.0, I run into [this issue](https://github.com/tensorflow/tensorflow/issues/27120). The very same notebook using TF 2.2 works, but I have found TF2.2 to be more or less a desaster, especially concerning performance (e.g., loading the saved model takes minutes as opposed to a few seconds with TF2.3). So, if you plan on supporting TF 2.3, that would be highly appreciated.

And: thanks for sharing your work!

**Comment by palatos** [comment]
Just out of curiosity, if you use this fork, does it solve the current error?
https://github.com/palatos/tf-explain

**Comment by dspyrhsu** [comment]
So, after doing this:
```
python -m pip install --upgrade git+https://github.com/palatos/tf-explain
[... snip ...]
Successfully installed tf-explain-0.2.1
```
The rror is gone! Thus, please write the pull request and thanks for your effort, @palatos !

**Comment by palatos** [comment]
@dspyrhsu I'm glad it worked! I was having the same issue and tried out a suggestion I saw someone else post, on that fork.
I think there is a similar pull request currently under review, but I'm glad this workaround is working for you too.

**Comment by RaphaelMeudec** [comment]
@dspyrhsu @palatos the problem is fixed on master, thanks for raising it! It will be shipped in a release next week after we merge #161
